functions/Get-DbaDbMailAccount.ps1

```diff
@@ -67,6 +67,7 @@ function Get-DbaDbMailAccount {
     param (
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
+        [Alias("Name")]
```

Comment:
Any reason the remove just uses `Account` instead of `Name`?

Reply:
Not sure I follow the question. The Remove- function uses `Name`, as the New- function, and to make it consistent with the *DbMailProfile functions that use `Name` instead of `$profile` which is an automatic variable.
Hence added alias `Name` to Get- function.

Reply:
See response on other PR but just because a variable is an "session variable" in PowerShell does not mean it cannot be used in functions. It has different scoping and will have no effect on a user's profile script path.

Reply:
Removed that alias, so it will be Account only (same for Profile for the other function).